fdbserver/Coordination.actor.cpp

```diff
@@ -221,7 +221,7 @@ ACTOR Future<Void> openDatabase(ClientData* db,
 	++(*clientCount);
 	hasConnectedClients->set(true);
 
-	if (req.supportedVersions.size() > 0) {
+	if (req.supportedVersions.size() > 0 && !req.internal) {
```

Comment:
Just note that this may result in wrong number of connected client reported by the status json. Basically internal client is no longer counted as client. The risk is that if we have large number of such clients, it won't be easy to identify.

Reply:
Sorry, didn't reply to this. Yes, valid point. The investigation is still in progress on this and a related clients issue (and it is possible we will have a follow up PR to this). We will revisit this after that investigation is done.